### ISSUE TITLE
Compiling the toolchain with MIPS will report an error and cannot be compiled in the past, I don't know what the reason is

### ISSUE THREAD

**Issue by hwk888** [heiher/hev-socks5-tunnel#57]
make[4]: Nothing to be done for 'static'.
make[4]: Nothing to be done for 'static'.
src/arch/mips/hev-task-execute-mips32.s: Assembler messages:
src/arch/mips/hev-task-execute-mips32.s:16: Error: invalid operands `lw $t0,($a0)'
src/arch/mips/hev-task-execute-mips32.s:17: Error: invalid operands `sw $sp,-0x4($t0)'
src/arch/mips/hev-task-execute-mips32.s:18: Error: invalid operands `sw $ra,-0x8($t0)'
src/arch/mips/hev-task-execute-mips32.s:19: Error: operand 2 must be an immediate expression `addiu $sp,$t0,-0x20'
src/arch/mips/hev-task-execute-mips32.s:24: Error: invalid operands `lw $ra,0x18($sp)'
src/arch/mips/hev-task-execute-mips32.s:25: Error: invalid operands `lw $sp,0x1c($sp)'
make[4]: *** [Makefile:84: build/kern/task/hev-task-execute.o] Error 1

**Comment by heiher** [comment]
Download toolchains from: https://musl.cc/

**Comment by hwk888** [comment]
thank you！Can I only compile with the toolchain for that URL? I can use other toolchains of the same architecture?

**Comment by hwk888** [comment]
I can use openwrt mips toolchain? Compiling the toolchain with MIPS by use openwrt toolchain

**Comment by heiher** [comment]
I don't know what toolchain you are using, but it looks like there is something wrong with the toolchain. I can confirm is that the toolchains of musl and openwrt can be compiled successfully, but the generated binary may not be compatible with your target system.

**Comment by hwk888** [comment]
yes,I'm using toolchain will report an error and cannot be compiled, my target system is mips of openwrt, Let me try again. 
This part of the assembly language can be dispensed with, or replaced by pure C

**Comment by hwk888** [comment]
ok
